src/components/Actions.jsx

```diff
@@ -12,18 +12,13 @@ const useStyles = makeStyles(theme => ({
     marginRight: 10,
     backgroundColor: '#364b8a',
     color: 'white'
-  },
-  row: {
-    flexDirection: 'row-reverse',
-    display: 'flex',
-    paddingTop: 10
   }
 }))
 
-export default function ButtonActions ({ okName, cancelName, onOk, onCancel }) {
+export default function ButtonActions ({ okName, cancelName, onOk, onCancel, style }) {
   const classes = useStyles()
   return (
-    <Box className={classes.row}>
+    <Box style={style}>
```

Comment:
You can assign the style with a condition, in case a custom style is passed assign that, otherwise use the one we already have.
Because the default style is already used in some components, with this change we have to pass the same style in multiple files, as you already did in this PR.

Something like style={ hasCustomStyle ? style: classes.row }
And hasCustomstyle refers to is the style is null or empty